modules/rosa-update-cli-tool.adoc

```diff
@@ -0,0 +1,17 @@
+
+// Module included in the following assemblies:
+//
+// * rosa-release-notes/rosa-release-notes.adoc
+:_content-type: REFERENCE
+[id="updating_rosa_cli{context}"]
+== Updating the ROSA CLI tool
+
+You can download the latest version of the ROSA CLI from the Hybrid Cloud Console.
+
+To update to the latest version of the ROSA CLI:
+
+. Download the file from the link:https://console.redhat.com/openshift/downloads[Hybrid Cloud Console].
+. Unzip the downloaded file.
+. In the terminal, type `sudo mv rosa /usr/bin/rosa` then press **Enter**.
```

Comment:
This really needs to be in `[source,terminal]` code block formatting because we are providing a command.